src/Microsoft.CodeAnalysis.Razor/DefaultTagHelperDescriptorFactory.cs

```diff
@@ -44,9 +31,8 @@ internal class DefaultTagHelperDescriptorFactory
                 .WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Omitted)
                 .WithMiscellaneousOptions(SymbolDisplayFormat.FullyQualifiedFormat.MiscellaneousOptions & (~SymbolDisplayMiscellaneousOptions.UseSpecialTypes));
 
-        public DefaultTagHelperDescriptorFactory(Compilation compilation, bool designTime)
+        public DefaultTagHelperDescriptorFactory(Compilation compilation)
```

Comment:
I think I know what I want to do with this now.

I want all of the providers to just do what they do regardless of runtime or design time. I want the tooling-specific API to just remove the ones that we're ignoring.

It's important that we don't break compat with this change 👍